metadata.json

```diff
@@ -8,21 +8,21 @@
   "project_page": "https://github.com/voxpupuli/puppet-etherpad",
   "issues_url": "https://github.com/voxpupuli/puppet-etherpad/issues",
   "dependencies": [
+    {
+      "name": "puppetlabs-concat",
+      "version_requirement": ">= 4.7.0 < 7.0.0"
+    },
     {
       "name": "puppetlabs-stdlib",
-      "version_requirement": ">= 4.17.0 < 6.0.0"
+      "version_requirement": ">= 4.25.0 < 7.0.0"
```

Comment:
if we bump the minimal version required, the PR needs to be marked as `backwards-incompatible`